settings.gradle

```diff
@@ -20,3 +20,4 @@ include ':camera-viewfinder-compose'
 include ':feature:settings'
 include ':data:settings'
 include ':core:common'
+include ':feature:quicksettings'
```

Comment:
is this needed for settings?

Reply:
Yes. Gradle build would fail without this